mne/viz/tests/test_raw.py

```diff
@@ -389,6 +396,9 @@ def test_plot_raw_ssp_interaction(raw, browser_backend):
     assert _proj_status(ssp_fig, browser_backend) == [True, True, True]
 
 
+# see https://github.com/mne-tools/mne-python/issues/10961
+@pytest.mark.skipif(
+    sys.platform.startswith('win'), reason='Needs fix on Windows')
```

Comment:
Yikes, it's all of these? I thought it was just `notebook` stuff that was broken

Reply:
Yeah it was 22 failed tests, I think about 12 were unique to parameterization. I have no idea why they are failing, I raised the issue in `ipympl` above because it seemed so odd. They work on Linux/MacOS so it seems like there is an underlying `ipympl` settings issue that causes odd things to happen. I can get on a `Windows` computer and have a look in 3 hours (in use right now).

Reply:
Yeah I assumed the `ipympl` stuff just meant that we would skip a couple of isolated `notebook` tests, but these are core viz tests. If `ipympl` installation breaks this many tests on Windows, we might be better off *uninstalling* `ipympl` from windows for now so we don't have to skip these tests.

Reply:
Hmm, okay, then `notebook` won't render `matplotlib` plots on `Windows` I think. I'll take a quick look at `ipympl` CIs and see if they test this kind of thing or what.

Reply:
Well that was quick, they don't have any CIs that run on `Windows` so I think that would be the issue. That really creates a headache for `notebook` support...

Reply:
> Hmm, okay, then notebook won't render matplotlib plots on Windows I think.

That's the case on `main` and I think we need to fix it separately. Let's just not install `ipympl` on Windows for now so we can move forward with this hopefully!

Reply:
I think the tests will pass without `ipympl` on `Windows` for this PR but I'm pretty sure the GUIs won't render properly. If we just do that in the CIs then the GUIs will work if someone installs via the `requirements.txt` file but these core viz tests fill fail and there could be more trouble. If we remove it from the `requirements.txt`, then maybe we should officially remove support for GUIs on `Windows` so that people know that it's not supposed to work. It's really bad that if someone just installs `ipympl` on `Windows`, the basic functionality of their MNE installation might break... That might be worth mentioning somewhere as well.

Reply:
> I think the tests will pass without ipympl on Windows for this PR 

Good!

> but I'm pretty sure the GUIs won't render properly...

This is all the case on `main`, correct?

Let me iterate again -- if it's the case on main, then please let's deal with it in different PR where we figure out how to fix things properly. Adopting the new abstraction framework -- which this PR moves us toward -- is separable from how things work on windows, *at least to the extent that things on windows are in the same state/probably broken with our current framework on main as well*.

If we try to tackle these two problems simultaneously, then it creates a review nightmare, and prevents any new progress with a new framework (that you advocated for so strongly!). We could tackle the ipympl-and-testing-on-windows first and then come back to this PR if you really want, but then this PR is stuck and you can't use your new framework.

Reply:
Yeah, this should all be broken on `main`, it's just coming up because to get things to work properly for the abstraction, you need `ipympl`. I just pushed a commit that hopefully should pass, is that what you were thinking?

Reply:
Yes that's more or less the idea, but don't forget to remove all the `skipif`s since we don't need them now

Reply:
They're all gone in the diff.

I left the one on brain scraper because it seems like it's actually a separate issue; here's the CI where it alone failed (https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=20714&view=logs&j=dded70eb-633c-5c42-e995-a7f8d1f99d91&t=8394206f-a00b-5cb7-44ca-464c9d5f9e25).